app/views/shared/_webchat.html.erb

```diff
@@ -34,4 +34,4 @@
   </span>
 </p>
 <% # This is inline in the source however slimmer will optimize this. %>
-<%= javascript_include_tag "webchat", integrity: false %>
+<%= javascript_include_tag "webchat", integrity: false, type: "module" %>
```

Comment:
Looks good to me as well, I tested one of the pages where webchat is used - https://government-frontend-pr-3282.herokuapp.com/government/organisations/hm-passport-office/contact/hm-passport-office-webchat

Still functions as expected and is included in the standardx checks.